Components/VatIdValidatorResult.php

```diff
@@ -297,8 +297,10 @@ private function init(string $namespace): void
 
     /**
      * Helper function to add an error by its error code
+     *
+     * @param string|int $errorCode
      */
-    private function addError(string $errorCode): void
+    private function addError($errorCode): void
```

Comment:
instead of removing a type from a parameter, it should be made sure, that the given code is always a string